src/Domain/Project/Project.entity.ts

```diff
@@ -20,13 +20,22 @@ export class Project {
   @Column({ type: 'timestamp', default: () => 'CURRENT_TIMESTAMP' })
   private createdAt: Date;
 
+  @Column({ type: 'boolean', nullable: false, default: false })
```

Comment:
C'est un détail mais par défaut on ne devrait pas créer des projects actifs ?

```suggestion
  @Column({ type: 'boolean', nullable: false, default: true })
```